yarn.lock

```diff
@@ -2189,23 +2189,6 @@
     ts-dedent "^2.0.0"
     util-deprecate "^1.0.2"
 
-"@storybook/addons@6.4.17":
```

Comment:
I don't understand why Dependabot is removing these gems from the lock file. @jibees, do you know?

Reply:
I don't understand at all too. This worth investigate before merging.

Reply:
Still removed...